src/plugins/discover/public/application/angular/doc_table/lib/get_default_sort.ts

```diff
@@ -26,12 +25,12 @@ import { SortOrder } from '../components/table_header/helpers';
  * the default sort is returned depending of the index pattern
  */
 export function getDefaultSort(
-  indexPattern: IndexPattern,
+  indexPattern: IndexPattern | undefined,
```

Comment:
In what scenario would index pattern be undefined?

Reply:
It shouldn't. But when using this function in an embeddable, according the TypeScript it could be. So the decision was the edit in the function or in the embeddable. decided to modify here, since the function would work would an index pattern.